tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -13,6 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+.red-dot {
+  background-color: #d93025;
+  border-radius: 5px;
+  height: 10px;
+  position: absolute;
```

Comment:
Is this component a `relative` container? If not, you may want to put one since it can wrongly place the red dot.